.github/workflows/terraform.yaml

```diff
@@ -0,0 +1,36 @@
+name: "Validate Terraform files"
+
+on:
+  push:
+    branches:
+      - "**"
+
+jobs:
+  terraform:
+    name: Terraform
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        directory: ["./terraform"]
```

Comment:
any reason to do matrix testing if there's only one directory?

Reply:
FYI: Just copy/paste. Took it out before merging.